commit 0be367943364272ecd5e4b1bfdf195d6eeca229c (refs/changes/60/4860/5)
Author: Mike Gerdts <mike.gerdts@joyent.com>
Date:   2018-09-24T18:00:10+00:00 (1 year ago)
    
    OS-7258 sysinfo needs to indicate if a PI can run kvm and bhyve concurrently
    Reviewed by: Josh Wilsdon <josh.wilsdon@joyent.com>
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/src/sysinfo b/src/sysinfo
index 106da2ba..f1dbf980 100644
--- a/src/sysinfo
+++ b/src/sysinfo
@@ -219,6 +219,7 @@ function get_bhyve_capability()
             if [[ -n ${vcpus} ]]; then
                 Bhyve_Max_Vcpus=${vcpus}
             fi
+            HVM_API="true"
         fi
     fi
 }
@@ -691,6 +692,7 @@ CPU_Virtualization='${CPU_Virtualization}'
 CPU_Physical_Cores=${CPU_Count}
 Bhyve_Capable='${Bhyve_Capable}'
 Bhyve_Max_Vcpus=${Bhyve_Max_Vcpus}
+HVM_API='${HVM_API}'
 Nic_Tags=${NicTagList}
 Admin_NIC_Tag=${Admin_NIC_Tag}
 Setup='${Setup_complete}'
@@ -826,6 +828,7 @@ END
   "VM Capable": ${VM_Capable},
   "Bhyve Capable": ${Bhyve_Capable},
   "Bhyve Max Vcpus": ${Bhyve_Max_Vcpus},
+  "HVM API": ${HVM_API},
   "CPU Type": "${CPU_Version}",
   "CPU Virtualization": "${CPU_Virtualization}",
   "CPU Physical Cores": ${CPU_Count},
